pkg/detectors/axonaut/axonaut.go

```diff
@@ -47,7 +48,8 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 		}
 
 		if verify {
-			req, err := http.NewRequestWithContext(ctx, "GET", "https://axonaut.com/api/v2/companies?type=all&sort=id", nil)
+			// Reference : https://axonaut.com/api/v2/doc
+			req, err := http.NewRequestWithContext(ctx, "GET", "https://axonaut.com/api/v2/me", nil)
```

Comment:
The API returns user email which we can add in extra data but the API also return some other sensitive information about user. I'll let @mcastorina @abmussani comment here.